docs/content/concepts/transactions/prog-txn-blocks.mdx

```diff
@@ -7,28 +7,228 @@ In basic terms, a blockchain is a block of transactions chained together in a sh
+A PTB can perform up to 1,024 unique operations in a single execution, whereas transactions on traditional blockchains would require 1,024 individual executions to accomplish the same result. The structure also promotes cheaper gas fees. The cost of facilitating individual transactions is always more than the cost of those same transactions blocked together in a PTB. The difference is not unlike the petrol you save by making a single trip to the supermarket versus driving back and forth for each bag of groceries you purchase.
 
-:::
+The remainder of this topic covers the semantics of the execution of the transaction commands. It assumes familiarity with the Sui object model and the Move language. For more information on those topics, see the following documents:
 
-In addition to the efficiency gains of blocking commands together in a single transaction, the structure also promotes cheaper gas fees. The cost of facilitating individual transactions is always more than the cost of those same transactions blocked together in a PTB. The difference is not unlike the petrol you save by making a single trip to the supermarket versus driving back and forth for each bag of groceries you purchase.
+- [Object model](../object-model.mdx)
+- [Move Concepts](../sui-move-concepts.mdx)
 
-## PTB inputs
+## Transaction type
 
-Inputs are values that you use as arguments to the commands in the PTB. Inputs can either be an object reference (either to an [owned object](../object-ownership/address-owned.mdx), an [immutable object](../object-ownership/immutable.mdx), or a [shared object](../object-ownership/shared.mdx)), or an encoded value (for example, an encoded string used as an argument to a Move call). 
+There are two parts of a PTB that are relevant to execution semantics. Other transaction information, such as the transaction sender or the gas limit, might be referenced but are out of scope. The PTB consists of two components:
 
-## PTB commands
+The inputs, `[CallArg]`, is a vector of arguments, either objects or pure values, that can be used in the transaction commands. The objects are either owned by the sender or are shared/immutable objects. The pure values represent simple Move values, such as `u64` or `String` values, which can be constructed purely by their bytes.
 
-There are multiple types of commands that you can use in a PTB, starting with implementations of common asset manipulation operations such as coin merging or splitting, and ending with the ability to call any on-chain Move function. You can chain typed outputs that your PTB commands receive and use them as typed inputs in subsequent commands in your PTB. With the ability to chain commands, you can develop a programmable transaction that performs a variety of operations on myriad object types in a single execution.
+The commands, `[Command]`, is a vector of transaction commands. The possible commands are:
```

Comment:
Getting to here, I am lost on what the difference is between the two docs in this PR...

Reply:
It's called plagiarism ;)